bin/annotation_workflow.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os
+import re
+from pathlib import Path
+
+# Function to print GitHub action notice
```

Comment:
In Python docstrings go within the body of a thing, like:

```
def foo():
    """
    Foo's docstring.
    """
```

Also generally prefer returning to printing from within functions, better to return things. E.g. here, probably better is:

```python
def github_action_notice(file, url):
    return f"::warning file={file},line=1::Annotation: {url}"

and then do all the printing when you call this function.

More importantly your line number looks hard coded to 1 -- that's not correct, the line needs to be the line where the specification property appears, otherwise your annotations won't be in the right spot.
```